apps/app/src/test/scala/org/lfdecentralizedtrust/splice/integration/tests/SvValidatorLicenseIntegrationTest.scala

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package org.lfdecentralizedtrust.splice.integration.tests
+
+import com.digitalasset.canton.logging.SuppressionRule
+import com.digitalasset.canton.topology.PartyId
+import org.lfdecentralizedtrust.splice.codegen.java.splice.validatorlicense.ValidatorLicense
+import org.lfdecentralizedtrust.splice.integration.EnvironmentDefinition
+import org.lfdecentralizedtrust.splice.sv.automation.delegatebased.MergeValidatorLicenseContractsTrigger
+import org.lfdecentralizedtrust.splice.util.TriggerTestUtil
+import org.lfdecentralizedtrust.splice.util.TriggerTestUtil.{
+  pauseAllDsoDelegateTriggers,
+  resumeAllDsoDelegateTriggers,
+}
+import org.slf4j.event.Level
+import scala.jdk.OptionConverters.*
+import scala.util.control.NonFatal
+
+class SvValidatorLicenseIntegrationTest extends SvIntegrationTestBase with TriggerTestUtil {
+
+  override def environmentDefinition: EnvironmentDefinition =
+    EnvironmentDefinition.simpleTopology1Sv(this.getClass.getSimpleName)
+
+  "grant validator license and verify merged licenses" in { implicit env =>
+    val dsoParty = sv1Backend.getDsoInfo().dsoParty
+    val sv1Party = sv1Backend.getDsoInfo().svParty
```

Comment:
```suggestion
    val info =  = sv1Backend.getDsoInfo();
    val dsoParty = info.dsoParty
    val sv1Party = info.svParty
```

not a problem here, but reading twice via RPC is a bit of an anti-pattern, as it leads to race conditions wrt inconsistent reads in case of mutable data.